### ISSUE TITLE
[Feature enhancement] Improve user templates

### ISSUE THREAD

**Issue by waahah** [Xdy1579883916/vanilla-pudding#8]
*It is hoped that `create-vpu` can be separated into a standalone demonstration project and the documentation can be improved.*

**Comment by Xdy1579883916** [comment]
Yes, I have such an intention. Currently, I only keep some API demonstrations in the template. It seems that the project is not very popular at present. Maybe I am the only one using it, so I haven't prepared relevant documents yet.
